### PR TITLE
Add new ofl fonts

### PR DESCRIPTION
**Pull request by AlexanderBelokon**
This PR adds some OFL fonts and updates `mapnik` to work with modern node

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 mapbox-studio-default-fonts
 ===========================
 
-Copyright information for each font is included in each folder in a file named `LICENSE.txt`.
+Copyright information for each font is included in each folder in a file named `LICENSE.txt`/`OFL.txt`/`UFL.txt`/`ofl.markdown`/`Open Font License.markdown`.
```
Comment (AlexanderBelokon) [inline README.md:4]: The licenses have different names and formats, might make sense to rephrase that to be more generic or alternatively rename all license files to `LICENSE.xxx`

**Review by tristen — APPROVED** [review]
Changes LGTM

File: bin/generate-font-families.js [hunk 2]
```diff
@@ -39,7 +38,13 @@ const keywords = [
     'dash'
 ];
 
-function getFontFamilies() {
+async function getFontFamilies() {
+    const dirs = (await fs.readdir(`./`))
+        .filter(d => !d.match(/^\.|node_modules/))
+    for (const dir of dirs) {
+        mapnik.register_fonts(path.join('./', dir), { recurse: true })
+    }
```
Comment (AlexanderBelokon) [inline bin/generate-font-families.js:46]: It's important not to load fonts that are not part of this package

File: font_families.json [hunk 3]
```diff
@@ -48,36 +64,6 @@
...
-  "DejaVu Serif": [
-    "DejaVu Serif Book",
-    "DejaVu Serif Bold",
-    "DejaVu Serif Condensed",
-    "DejaVu Serif Condensed Bold",
-    "DejaVu Serif Italic",
-    "DejaVu Serif Bold Italic",
-    "DejaVu Serif Condensed Italic",
-    "DejaVu Serif Condensed Bold Italic"
-  ],
```
Comment (AlexanderBelokon) [inline font_families.json:80]: These fonts are not actually available in Studio

File: package.json [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
   "main": "index.js",
   "scripts": {
     "build": "node bin/generate-font-families.js"
   },
   "dependencies": {
-    "mapnik": "^3.6.2"
+    "mapnik": "^4.5.9"
+  },
+  "devDependencies": {
+    "node-fetch": "^3.3.1"
```
Comment (AlexanderBelokon) [inline package.json:13]: Needed to upload fonts to the api

**Pull request merged**